### DISCUSSION TITLE
Why wont the game let me put any inputs in?

### DISCUSSION THREAD

**Discussion by IMDAHBIGBOSS** [visualboyadvance-m/visualboyadvance-m#1166] [Q&A] [answered]
After 9 grogrillion years I managed to get the emulator game running, though now it wont let me do anything. I tried rearranging the inputs but it didn't help. How do I fix this?

**Comment by ZachBacon** [comment, marked as answer]
Use something that isn't a Chromebook. It's a little outside our scope.

**Reply by IMDAHBIGBOSS** [reply]
Could you explain why a chromebook wont work in layman terms. If you do I can probably find an extension or something that will fix this.

**Reply by ZachBacon** [reply]
> Could you explain why a chromebook wont work in layman terms. If you do I can probably find an extension or something that will fix this.

Simply put ChromeOS isn't something we use. It's just wasn't in our scope. So it would take time to deal with how ChromeOS does things.
